src/components/Staffer/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from '@edx/paragon';
+
+
+const getStafferName = staffer => `${staffer.given_name} ${staffer.family_name || ''}`;
+
+const Staffer = ({ staffer, onRemove }) => (
+  <div className="staffer-wrapper col-6 col-md-4 col-lg-3 col-xl-2" key={staffer.uuid}>
+    <div className="staffer-image-wrapper overflow-hidden">
+      <img src={staffer.profile_image_url} className="rounded-circle w-50" alt="" />
+    </div>
+    <div className="staffer-details">
+      <button className="btn js-delete-btn mr-1 p-0" onClick={() => onRemove(staffer.uuid)}>
```

Comment:
I can see an argument for moving this out of staffer and having it on the actual grid, so I'm open to whatever

Reply:
In the staffer makes sense to me.